sdk/cosmos/azure_data_cosmos/README.md

```diff
@@ -1,5 +1,200 @@
 # Azure Cosmos DB SDK for Rust.
 
-## Introduction
+This client library enables client applications to connect to Azure Cosmos DB via the NoSQL API. Azure Cosmos DB is a globally distributed, multi-model database service.
 
-This client library enables client applications to connect to [Azure Cosmos DB](https://azure.microsoft.com/en-us/products/cosmos-db) via the NoSQL API. Azure Cosmos DB is a globally distributed, multi-model database service.
+[Source code] | [Package (crates.io)] | [API reference documentation] | [Azure Cosmos DB for NoSQL documentation]
+
+## Getting started
+
+### Install the package
+
+Install the Azure Cosmos DB SDK for Rust with cargo:
+
+```bash
```

Comment:
Nit: we've been trying to standardize on `sh` in general, but `azurecli` for `az`-specific command lines. IIRC, that actually does some magic on Microsoft Learn, though I'm not sure how many of these READMEs get up there in a way where that helps.